_rules/table-header-cell-has-assigned-cells-d0f69e.md

```diff
@@ -240,10 +247,10 @@ This `div` with role of `columnheader` and text equal to "Occupant" does not hav
 		<div role="columnheader">Occupant</div>
 	</div>
 	<div role="row">
-		<div role="gridcell">1A</div>
+		<div role="gridcell"><button>1A</button></div>
```

Comment:
I'm not sure why we need to add `button` here (and in the next row).

Reply:
This was not meant to be a failed example.